admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DocumentTypeServiceImpl.java

```diff
@@ -428,6 +440,17 @@ public DocumentTypeResponseDto getAllDocumentTypeByLaguageCode(String langCode)
 		return documentTypeResponseDto;
 	}
 
+	private void validateLangCode(String langCode) {
+		if (langCode == null || langCode.trim().isEmpty()) {
+			throw new IllegalArgumentException("Language code cannot be null or empty");
```

Comment:
please add the error code

Reply:
Added proper error codes for empty language scenarios as 
throw new DataNotFoundException(LanguageErrorCode.NO_LANGUAGE_FOUND_EXCEPTION.getErrorCode(),
					LanguageErrorCode.NO_LANGUAGE_FOUND_EXCEPTION.getErrorMessage())